### PR TITLE
changed: added setting for use of video tags

### PR DESCRIPTION
**Pull request by notspiff**
Default to false.

Too much garbage in the tags in files out there..

### CODE REVIEW

**Review by MartijnKaijser — APPROVED** [review]
Thanks :)

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -13574,7 +13574,17 @@ msgctxt "#21342"
...
 
-#empty strings from id 21343 to 21357
+#: system/settings.xml
+msgctxt "#21343"
+msgid "Use video tags"
+msgstr ""
+
+#: system/settings.xml
+msgctxt "#21344"
+msgid "If mp4 or mkv files have tags, use this to for library metadata"
```
Comment (ksooo) [inline addons/resource.language.en_gb/resources/strings.po]: to -> too ?

File: xbmc/video/tags/VideoInfoTagLoaderFactory.cpp [hunk 2]
```diff
@@ -38,7 +40,8 @@ IVideoInfoTagLoader* CVideoInfoTagLoaderFactory::CreateLoader(const CFileItem& i
     return nfo;
   delete nfo;
 
-  if (item.IsType(".mkv") || item.IsType(".mp4"))
+  if (CServiceBroker::GetSettings().GetBool("myvideos.usetags") &&
```
Comment (ksooo) [inline xbmc/video/tags/VideoInfoTagLoaderFactory.cpp]: We usually add a string constant for settings here: https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.h + https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.cpp

**Comment by ksooo** [comment]
@MartijnKaijser approved? Not that fast, please... ;-)

**Review by ksooo — CHANGES_REQUESTED** [review]
;-)

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (notspiff) [reply addons/resource.language.en_gb/resources/strings.po]: Ehrm 'to fill' or 'for' and i ended up with useless hybrid.

File: xbmc/video/tags/VideoInfoTagLoaderFactory.cpp (hunk 2 above)
Comment (notspiff) [reply xbmc/video/tags/VideoInfoTagLoaderFactory.cpp]: I guess i will have to comply with this horrid idea ;)

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (ksooo) [reply addons/resource.language.en_gb/resources/strings.po]: hehe

Comment (notspiff) [reply addons/resource.language.en_gb/resources/strings.po]: removed to 'to'

File: xbmc/video/tags/VideoInfoTagLoaderFactory.cpp (hunk 2 above)
Comment (notspiff) [reply xbmc/video/tags/VideoInfoTagLoaderFactory.cpp]: added.

**Review by ksooo — APPROVED** [review]

**Comment by Memphiz** [comment]
That was quick ;)

**Comment by notspiff** [comment]
Martijn excels at whining... ;)

3. jan. 2018 21:34 skrev "Memphiz" <notifications@github.com>:

**Pull request merged**